packager/media/codecs/h264_byte_to_unit_stream_converter_unittest.cc

```diff
@@ -108,20 +108,43 @@ TEST(H264ByteToUnitStreamConverter, NaluConversionWithSpsExtension) {
       0x00, 0x01, 0x68, 0xFE, 
       0x00, 0x00, 0x00, 0x01, 	// Start code
       // SPS Extension data
-      0x00, 0x05, 0x6d, 0x33, 0x01, 0x57, 0x78
+      0x00, 0x05, 0x6D, 0x33, 0x01, 0x57, 0x78
   };
 
-  int kSpsExtSize = arraysize(kSpsExtArr);
-  std::vector<uint8_t> kSpsExt(kSpsExtArr, kSpsExtArr + kSpsExtSize);
+  std::vector<uint8_t> byte_stream_with_sps_extension(std::begin(kByteStreamWithSpsExtension), 
+  						      std::end(kByteStreamWithSpsExtension));
```

Comment:
Can you keep the length of the line within 80 characters?

Reply:
fixed

Reply:
Also, when I open the file h264_byte_to_unit_stream_converter.cc from the most recent commit, I don't see the error of ParseSpsFromBytes not aligning with const. Maybe it didn't update the changelist, but the file was updated?